### PR TITLE
Fix #358. Document ChronicleInit- support ticket 569.

### LINKED ISSUE #358: Document support ticket 569 - Jvm hook
@foroughgoudarzi please add a description

### PR DESCRIPTION
**Pull request by foroughgoudarzi**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -57,6 +57,12 @@ See `Jvm.Main` in the `src/test` as an example.
 NOTE: If you specify system properties on the command line then they will override those in the
 `system.properties` file.
 
+== ChronicleInit
+
+The link:https://github.com/OpenHFT/Chronicle-Core/blob/ea/src/main/java/net/openhft/chronicle/core/ChronicleInit.java[`ChronicInit`] class handles application code which must be loaded/run before any system property is read and may override system properties.
```
Comment (rogersimmons) [inline README.adoc]: Typo? ChronicleInit (le missing)

**Review by rogersimmons — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (rogersimmons) [reply README.adoc]: (reset approval)

**Review by rogersimmons — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (JerryShea) [reply README.adoc]: "handles application code" maybe say something like "allows the developer to hook in their own code to be run at startup before the Chronicle static initialisers are run. Chronicle static initialisers do things like load system properties"

**Review by JerryShea — APPROVED** [review]
perfect!

**Review by rogersimmons — APPROVED** [review]

**Pull request merged**